SCClassLibrary/Common/GUI/Base/Grid.sc

```diff
@@ -102,7 +119,7 @@ DrawGridX {
 		^commands ?? {
 			cacheKey = [range,bounds];
 			commands = [];
```

Comment:
It originally was an array and I kept it at that because adding things to the `Array` this way is faster than adding to the `List`. Since this is ultimately called from the draw loop, I think that optimization should stay, even though it doesn't look good...